### ISSUE TITLE
[BUG] H025 false positive

### ISSUE THREAD

**Issue by jayvdb** [djlint/djLint#90]
- [x] I'm on the [latest version](https://pypi.org/project/djlint/) of djLint
- [x] I've searched the [issues](https://github.com/Riverside-Healthcare/djLint/issues)
- [x] I've read the [docs](https://djlint.com)

## System Info
 - OS: Windows
 - Python Version: 3.7
 - djLint Version: 0.5.6

## Issue
The whitespace in `<tr >` causes `H025 Tag seems to be an orphan. </tr>`:

```html
                <tr >
                    <td>Foo</td>
                </tr>
```
